src/main/java/matcha/exception/MatchaException.java

```diff
@@ -0,0 +1,24 @@
+package matcha.exception;
+
+/**
+ * Represents a custom exception for all errors related to the Matcha program
+ */
+public class MatchaException extends Exception {
```

Comment:
Should there be a more specific Exception to catch to avoid Pokemon Exception Handling?